trade-finance-manager-ui/templates/utilisation-reports/_macros/premium-payments-table.njk

```diff
@@ -3,8 +3,109 @@
 {% import "../../_macros/table-cell-checkbox.njk" as tableCellCheckbox %}
 {% import "../../_macros/select-all-table-cell-checkbox.njk" as selectAllTableCellCheckbox %}
 
+{% macro renderPremiumPaymentsTableRow(params) %}
+  {% set feeRecordPaymentGroup = params.feeRecordPaymentGroup %}
+
+  {% set feeRecords = feeRecordPaymentGroup.feeRecords %}
+  {% set totalReportedPayments = feeRecordPaymentGroup.totalReportedPayments %}
+  {% set paymentsReceived = feeRecordPaymentGroup.paymentsReceived %}
+  {% set totalPaymentsReceived = feeRecordPaymentGroup.totalPaymentsReceived %}
+  {% set status = feeRecordPaymentGroup.status %}
+  {% set displayStatus = feeRecordPaymentGroup.displayStatus %}
+  {% set checkboxId = feeRecordPaymentGroup.checkboxId %}
+  {% set isChecked = feeRecordPaymentGroup.isChecked %}
+
+  {% for feeRecord in feeRecords %}
+    {% set isFirstRow = loop.index === 1 %}
+    {% set isLastRow = loop.index === feeRecords.length %}
```

Comment:
I know this is html rows within conceptual rows but to keep confusion to a minimum perhaps we could name these 'isFirstFeeRecordInGroup' and 'isLastFeeRecordInGroup' instead?